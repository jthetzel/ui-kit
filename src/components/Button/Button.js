import React from 'react';
import styled from 'react-emotion';

const Button = styled.button`
  background-color: salmon;
  border: 1px solid indianred;
  color: snow;
`;

/**
 * @component
 */
export default props => <Button {...props} />;
