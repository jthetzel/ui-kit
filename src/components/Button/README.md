Button

```js
<Button>Push Me</Button>
```