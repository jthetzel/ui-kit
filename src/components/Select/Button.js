import styled from 'react-emotion';

export default styled.div`
  position: relative;
  border-top: 1px solid black;
  border-bottom: 1px solid black;
  padding: 20px 0;
  user-select: none;
  width: 100%;
  
  &:hover {
    cursor: pointer;
  }
`;
