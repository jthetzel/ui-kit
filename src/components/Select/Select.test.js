import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Button from '../Button';

test('Button does not crash', () => {
  const tree = shallow(<Button>title</Button>);
  expect(toJson(tree)).toMatchSnapshot();
});
