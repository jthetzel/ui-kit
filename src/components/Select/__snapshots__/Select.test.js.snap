// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Select does not crash 1`] = `
<DropdownWrapper
  isOpen={false}
  opener={[Function]}
>
  <Styled(div)
    onClick={[Function]}
  >
    <List
      onChange={[Function]}
      options={
        Array [
          Object {
            "label": "One",
            "value": "one",
          },
          Object {
            "label": "Two",
            "value": "two",
          },
          Object {
            "label": "Three",
            "value": "three",
          },
          Object {
            "label": "Four",
            "value": "four",
          },
        ]
      }
      value="one"
    />
  </Styled(div)>
</DropdownWrapper>
`;

exports[`Select simulate click Button 1`] = `
<DropdownWrapper
  isOpen={false}
  opener={[Function]}
>
  <Styled(div)
    onClick={[Function]}
  >
    <List
      onChange={[Function]}
      options={
        Array [
          Object {
            "label": "One",
            "value": "one",
          },
          Object {
            "label": "Two",
            "value": "two",
          },
          Object {
            "label": "Three",
            "value": "three",
          },
          Object {
            "label": "Four",
            "value": "four",
          },
        ]
      }
      value="one"
    />
  </Styled(div)>
</DropdownWrapper>
`;
